2226/2/data_storage.rb

```diff
@@ -0,0 +1,8 @@
+# This class sis used for storing data
+class DataStorage
+  def self.show_all_data
```

Comment:
create an instance and use it. And you can pass the path to the folder into `#new`.